<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogExportParams</class>
 <widget class="QDialog" name="DialogExportParams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Specify video parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupVideo">
     <property name="title">
      <string>Video encoding parameters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Image size:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="boxImageSize">
          <property name="whatsThis">
           <string>Specifies the output image size. Note that some output formats (VideoCD, DVD, HD) require a specific image size.</string>
          </property>
          <item>
           <property name="text">
            <string>352x288 (VCD PAL)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>352x240 (VCD NTSC)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>720x576 (DVD PAL)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>720x480 (DVD NTSC)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1280x720 (HD 720p)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1980x1080 (HD 1080p)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_3">
          <property name="whatsThis">
           <string>Which codec to use for video encoding</string>
          </property>
          <property name="text">
           <string>Video encoding:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="2">
         <widget class="QComboBox" name="boxVideoEncoding">
          <item>
           <property name="text">
            <string>xvid</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Frame rate:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="2">
         <widget class="QComboBox" name="boxFPS">
          <property name="whatsThis">
           <string>Output frame rate. PAL frame rate is 25 FPS, NTSC is 29.95 FPS</string>
          </property>
          <item>
           <property name="text">
            <string>25 FPS (PAL)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>29.97 FPS (NTSC)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Container format:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QComboBox" name="boxContainer">
          <property name="whatsThis">
           <string>Output file (container) format, i.e. AVI, MKV, MP4</string>
          </property>
          <item>
           <property name="text">
            <string>AVI</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MKV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MP4</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="cbAllKeyframes">
          <property name="whatsThis">
           <string>Export every video frame as keyframe. It produces MUCH larger video files, but the quality would be the best. Use if you are going to process the video further.</string>
          </property>
          <property name="text">
           <string>Every frame is a keyframe</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbExportNoAudio">
          <property name="whatsThis">
           <string>Export every video frame as keyframe. It produces MUCH larger video files, but the quality would be the best. Use if you are going to process the video further.</string>
          </property>
          <property name="text">
           <string>Do not export audio</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupCDG_2">
     <property name="title">
      <string>Lyrics rendering</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Font:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFontComboBox" name="fontVideo">
          <property name="whatsThis">
           <string>Font family for preview window</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>Size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="fontVideoSize">
          <property name="whatsThis">
           <string>Font size for preview window. Larger values are suggested (24 and up)</string>
          </property>
          <property name="minimum">
           <number>4</number>
          </property>
          <property name="maximum">
           <number>120</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDetectSize">
          <property name="text">
           <string>Detect size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPreview">
          <property name="text">
           <string>Preview</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Keep title at least </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="titleVideoMin">
          <property name="maximum">
           <number>120</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>seconds</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string>Colors</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>Backround</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ColorButton" name="btnVideoColorBg">
           <property name="whatsThis">
            <string>Preview window background color.</string>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>Information</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="ColorButton" name="btnVideoColorInfo">
           <property name="whatsThis">
            <string>Preview window text color for the text which has been sung.</string>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>Sung</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ColorButton" name="btnVideoColorInactive">
           <property name="whatsThis">
            <string>Preview window text color for the text which is not being sung yet..</string>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_31">
           <property name="text">
            <string>Not sung yet</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="ColorButton" name="btnVideoColorActive">
           <property name="whatsThis">
            <string>Preview window text color for the text which has been sung.</string>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbVideoPreamble">
        <property name="text">
         <string>Draw squares showing where singing starts</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Output file</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblOutput">
        <property name="text">
         <string>Write video to file:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leOutputFile"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnBrowse">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogExportParams</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogExportParams</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
